spinnaker_camera_driver/launch/driver_node.launch.py

```diff
@@ -108,20 +109,27 @@
 
 def launch_setup(context, *args, **kwargs):
     """Launch camera driver node."""
+    parameter_path = LaunchConfig('set_parameter_file').perform(context)
```

Comment:
Can we have a better name for "parameter_path" haha. parameter_path is almost the same as parameter_file. Otherwise this looks good